src/state/marketplace/hooks.tsx

```diff
@@ -163,18 +169,32 @@ export const useAddToMarketplace = (): (() => void) => {
   const nft = useNFT()
   const auction = useAuction()
 
-const createAuction = useCreateAuction()
-/** */
-  return useCallback(async() => {
-    if (seller && chainId && auction && nft) {
-      console.log(auction,'auction');
-     const tokenId=nft?.tokenId?nft?.tokenId:1;
+  const createAuction = useCreateAuction()
 
-      await createAuction( auction.contractAddress, tokenId,  auction.minBid as number,   auction.qualifyAmount as number,   auction.isForBid,auction.listingPrice as number,
-        auction.expireTimestamp)
-      dispatch(addToMarketplaceAction({ ...auction, nft: nft.id, seller, listTime: new Date(), chainId }))
+  useMarketplaceListener(nft)
+  return useCallback(async () => {
+    if (seller && chainId && auction && nft) {
+      const tokenId = nft?.tokenId ? nft?.tokenId : 1
```

Comment:
we cannot create with tokenId : 1

Reply:
We don't use tokenId for now I can delete it as we use the nft ID.